Marlin/src/feature/spindle_laser.cpp

```diff
@@ -67,7 +67,7 @@ cutter_frequency_t SpindleLaser::frequency;                           // PWM fre
 void SpindleLaser::init() {
   #if ENABLED(SPINDLE_SERVO)
     servo[SPINDLE_SERVO_NR].move(SPINDLE_SERVO_MIN);
-  #else
```

Comment:
This looks wrong to me, the #else is change to if and the remainder of the conditions are skipped when an ENA pin is not defined.

Reply:
Never mind, did not think it through.

Reply:
I would recommend something like this in conditionals

conditionals_adv.h
```
  #if PIN_EXISTS(SPINDLE_LASER_ENA)
    #define HAS_SPINDLE_LASER_ENA 1
  #endif
```
Then use TERN_ elsewhere e.g.

```TERN_(HAS_SPINDLE_LASER_ENA, OUT_WRITE(SPINDLE_LASER_ENA_PIN, !SPINDLE_LASER_ACTIVE_STATE));    // Init spindle to off```